src/cosmic_ray/tools/survival_rate.py

```diff
@@ -8,16 +10,65 @@
 def format_survival_rate():
     """cr-rate
 
-    Usage: cr-rate <session-file>
+Usage: cr-rate [--estimate] [--confidence <confidence_rate>] [--fail-over <max_value>] <session-file>
 
-    Calculate the survival rate of a session.
-    """
+Calculate the survival rate of a session.
+
+options:
+    --estimate                     Print the lower bound, estimate and upper bound of
+                                   survival rate
+    --confidence <confidence_rate> Specify the confidence levels for estimates, 95 (%) by
+                                   default. 80, 90, 95, 98, 99, 99.5, 99.8 and 99.9 are
+                                   supported.
+    --fail-over <max_value>        Exit with a non-zero code if the survival rate is
+                                   larger than <max_value> or the calculated confidence
+                                   interval is above the <max_value> (if --estimate is used).
+                                   Specified as percentage.
+"""
     arguments = docopt.docopt(
         format_survival_rate.__doc__, version='cr-rate 1.0')
+    show_estimate = arguments['--estimate']
+    confidence = arguments['--confidence']
+    fail_over = arguments['--fail-over']
+
+    # use integers as keys as equality is not well defined for floats
+    # the values are z-values for Standard Normal Probabilities
+    supp_z_scores = {800: 1.282,
+                     900: 1.645,
+                     950: 1.960,
+                     980: 2.326,
+                     990: 2.576,
+                     995: 2.807,
+                     998: 3.080,
+                     999: 3.291}
+
+    if not confidence:
```

Comment:
I know a confidence of zero is unlikely, but if someone supplied it for some reason then this would set it to 95. Should this check be `if confidence is None`?

Reply:
`confidence` here is a string, isn't it? so a "0" would evaluate to True

an empty string would cause it to default to 95, and I think that's acceptable